plugins/modules/mso_schema_template.py

```diff
@@ -130,6 +130,8 @@ def main():
     tenant = module.params.get('tenant')
     schema = module.params.get('schema')
     template = module.params.get('template')
+    if template is not None:
+        template = template.replace(' ', '')
```

Comment:
I would put the replace on line 160 like the other occurrences

Reply:
The template is not required, so we need to check whether the template is none before replacing.